lib/improver/tests/utilities/test_rescale.py

```diff
@@ -99,5 +99,97 @@ def test_clip(self):
         self.assertArrayAlmostEqual(result, expected)
 
 
+class Test_apply_double_scaling(IrisTest):
+
+    """Test the apply_double_scaling method."""
+
+    def setUp(self):
+        """Create cubes with a single zero prob(precip) point."""
+        self.cube_a = add_forecast_reference_time_and_forecast_period(
+            set_up_cube_with_no_realizations(zero_point_indices=[]),
+            fp_point=0.0)
+        self.cube_b = add_forecast_reference_time_and_forecast_period(
+            set_up_cube_with_no_realizations(zero_point_indices=[]),
+            fp_point=0.0)
```

Comment:
What are the data shapes and values in these cubes?

Reply:
Doc-string updated.